### ISSUE TITLE
Confirmation banner link hover colour

### ISSUE THREAD

**Issue by AdamChrimes** [nhsuk/cookie-consent#76]
After accepting the cookie consent on the NHS.UK website, I receive a confirmation banner which has a link to 'Turn cookies on or off'. The hover colour for this link is difficult to read and doesn't meet the AA accessibility requirements.

<img width="1679" alt="Screenshot 2019-06-20 at 14 38 45" src="https://user-images.githubusercontent.com/11615501/59853709-56493c00-9369-11e9-8af7-7e829f1bd5ed.png">

<img width="215" alt="Screenshot 2019-06-20 at 14 42 12" src="https://user-images.githubusercontent.com/11615501/59853867-a7f1c680-9369-11e9-817c-c99085932115.png">

We also had this issue on the NHSX website, but we hot-fixed it with some custom CSS, which you can find here: https://github.com/nhsx/nhsx-website/blob/develop/app/styles/nhsx/_utilities.scss#L66

But this should probably be fixed centrally as I imagine other websites have similar issues.
